src/openforms/formio/utils.py

```diff
@@ -221,3 +221,30 @@ def conform_to_mask(value: str, mask: str) -> str:
             )
 
     return "".join(result)
+
+
+def is_visible_in_frontend(component: dict, data: dict) -> bool:
```

Comment:
```suggestion
def is_visible_in_frontend(component: JSONObject, data: DataMapping) -> bool:
```
these type hints are more precise (both available in `openforms.typing`)